text/0061-error-format-and-definitions.md

```diff
@@ -122,6 +122,80 @@ HTTP Code: `400 Bad Request`
 
 ---
 
+## api_key_already_exists
+
+`Synchronous`
+
+### Context
+
+This error happens when a user tries to create an API Key that already exists for the given `uid`.
+
+### Error Definition
+
+HTTP Code: `409 Conflict`
+
+```json
+{
+    "message": "`uid` field value `:value` already exists for an API key.",
+    "code": "api_key_already_exists",
+    "type": "invalid_request",
+    "link": "https://docs.meilisearch.com/errors#api_key_already_exists"
+}
+```
+
+- The `:value` is inferred when the message is generated.
+
+---
+## invalid_api_key_uid
+
+`Synchronous`
+
+### Context
+
+This error happens when the `uid` field for an `API Key` resource is invalid.
+
+### Error Definition
+
+HTTP Code: `400 Bad Request`
+
+```json
+{
+    "message": "`uid` field value `:value` is invalid. It should be a valid uuidv4 string or ommited.",
```

Comment:
related to https://github.com/meilisearch/meilisearch/pull/2438#discussion_r885546155

```suggestion
    "message": "`uid` field value `:value` is invalid. It should be a valid UUID v4 string or omitted.",
```